spec/aws/action_mailer/ses_v2/mailer_spec.rb

```diff
@@ -33,12 +33,13 @@ module SESV2
         end
 
         before do
-          ::ActionMailer::Base.add_delivery_method :ses_v2, Mailer, **client_options
+          ::ActionMailer::Base.add_delivery_method :ses_v2, Mailer
```

Comment:
These might already be registered now with the railtie. The tests will require rails and this gem.

Reply:
They were not because the Railtie only executes in the context of a Rails application and only if the application is initialized. To ensure we're also testing the Railtie I added a stub Rails application in the `spec_helper` to force the initialization process to occur and the Railtie to register the delivery methods such that we can remove the `add_delivery_method` calls in the tests.